app/models/frontend/FrontendAnswers.scala

```diff
@@ -16,10 +16,11 @@
 
 package models.frontend
 
-import models.connector.api_1802.request.AnnualAllowances
+import models.connector.api_1802.request.{AnnualAdjustments, AnnualAllowances}
 
 abstract class FrontendAnswers[A] {
-  def toDownStream(current: Option[AnnualAllowances]): AnnualAllowances
+  def toDownStreamAnnualAllowances(current: Option[AnnualAllowances]): AnnualAllowances
+  def toDownStreamAnnualAdjustments(current: Option[AnnualAdjustments]): AnnualAdjustments = current.getOrElse(AnnualAdjustments.empty)
```

Comment:
These are changes that will be important for my other PR to correct how capital allowances are being saved https://github.com/hmrc/income-tax-self-employment/pull/149